src/main.js

```diff
@@ -177,6 +180,8 @@ function configureCommandlineSwitchesSync(cliArgs) {
 	// Read argv config
 	const argvConfig = readArgvConfigSync();
 
+	let hasLocaleSwitch = false;
```

Comment:
Rather than storing `hasLocaleSwitch`, I would suggest to store the locale from the `argv.json` if any and then have a single place where we call `app.commandLine.appendSwitch('lang', localeToUse);` in the end. I also think we can do the check for insiders/exploration there as well as for the pseudo language.